src/shogun/regression/LeastAngleRegression.cpp

```diff
@@ -105,7 +105,7 @@ void CLeastAngleRegression::plane_rot(ST x0, ST x1,
 template <typename ST, typename U>
 bool CLeastAngleRegression::train_machine_templated(CDenseFeatures<ST>* data)
 {
-	std::vector<std::vector<ST>> m_beta_path_t;		
+	std::vector<SGVector<ST>> m_beta_path_t;
```

Comment:
what is the reason for this? just asking :)

Reply:
I have made this change to observe more easily the `m_beta_path` variable. Since it was an `std::vector` I was forced to convert it to `SGVector` before using `observe` and that was not nice.

Reply:
ok thx. I think that is fine. maybe even make more sense as the inner vectors are of fixed size